sass/site/site.scss

```diff
@@ -158,7 +158,7 @@ $paddingNormal: 20px;
 // --------------------------------------------------
 .navbar {
   margin-bottom: 0px;
-  border-top: 3px solid $blueDark;
+  border-top: 3px solid #0d94d2; //cerner blue
```

Comment:
@jfewins @jon2992 added cernerBlue and cernerGreen to variable.scss https://github.com/sujaysudheenda/cerner.github.io/commit/afef62a86c17c8cf17090a94135dec4318a4d92d